scheduler/clients/javascript/tests/account.spec.ts

```diff
@@ -14,17 +14,23 @@ describe("Account API", () => {
       code: CREATE_ACCOUNT_CODE,
     });
     expect(status).toBe(201);
-    expect(data!).toBeDefined();
+    expect(data).toBeDefined();
   });
 
   it("should find account", async () => {
     const { status, data } = await client.account.create({
       code: CREATE_ACCOUNT_CODE,
     });
-    const accountClient = NettuClient({ apiKey: data!.secretApiKey });
+    if (!data) {
+      throw new Error("Account not created");
+    }
```

Comment:
I wonder if the lib should throw an error when something is not found instead of allowing `undefined` 🤔 (+ remove status & throw if status is not 20x)